packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -217,11 +217,11 @@ export class HostedPluginProcess implements ServerPluginRunner {
         }
         this.logger.error(`[${serverName}: ${pid}] IPC exited, with signal: ${signal}, and exit code: ${code}`);
 
-        const message = 'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page.';
-        let hintMessage: string = 'If it doesn\'t help, please check Theia server logs.';
+        const message = nls.localize('theia/plugin-ext/runtimeErrorMessage', 'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page.');
+        let hintMessage: string = nls.localize('theia/plugin-ext/runtimeErrorHintMessage', 'If it doesn\'t help, please check Theia server logs.');
         if (signal && signal.toUpperCase() === 'SIGKILL') {
             // May happen in case of OOM or manual force stop.
-            hintMessage = 'Probably there is not enough memory for the plugins. ' + hintMessage;
+            hintMessage = nls.localize('theia/plugin-ext/runtimeErrorAddHingMessage', 'Probably there is not enough memory for the plugins. ') + hintMessage;
```

Comment:
minor - typo:

```suggestion
            hintMessage = nls.localize('theia/plugin-ext/runtimeErrorAddingMessage', 'Probably there is not enough memory for the plugins. ') + hintMessage;
```

---

I believe we can simplify to `runtimeMemoryError`,